spec/ruby/core/io/close_on_exec_spec.rb

```diff
@@ -45,8 +45,16 @@
       -> { @io.close_on_exec = true }.should raise_error(IOError)
     end
 
-    it "returns nil" do
-      @io.send(:close_on_exec=, true).should be_nil
+    it "returns given argument" do
+      truthy = Object.new
+      (@io.close_on_exec = truthy).should equal(truthy)
+    end
```

Comment:
This is a spec of assignment expressions, but not of `IO#close_on_exec=` method.

Reply:
Yeah... I l have learned it from https://github.com/ruby/ruby/pull/4321#issuecomment-806224478, https://github.com/ruby/ruby/pull/4321#issuecomment-806302656 🙇 

Sorry to forget updating. I have added commit as https://github.com/ruby/ruby/pull/4321/commits/2c934b80330733f5cf92617248c00ea083feee9e